theme/migrations/0013_auto_20180222_1700.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('theme', '0012_1_15_RC_merge'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='userprofile',
+            name='email_opt_out',
+            field=models.BooleanField(default=False),
+        ),
+        migrations.AlterField(
```

Comment:
What's this model change? siteconfig.copyright?

Reply:
I didn't change the model.  This happened because of the limbo state the develop branch was in before getting the RC merged back in.  I'll back this out and submit another pull request with develop merged into this branch.

Reply:
Here is the commit with the change.
https://github.com/hydroshare/hydroshare/commit/039dc67d56e7da5a7f573a98f5b8db62839f00a3

Looks like a migration was never made after that change and I picked it up when I ran makemigrations for the userprofile model change.

Reply:
@aphelionz - are we good to go with this review then?

Reply:
This is fine, yeah